worlds/hk/Items.py

```diff
@@ -14,6 +15,9 @@ class HKItemData(NamedTuple):
     item_table[item_name] = HKItemData(advancement=item_name in logic_items or item_name in item_effects,
                                        id=i, type=item_type)
 
+for item_name in godhome_event_names:
+    item_table[item_name] = HKItemData(advancement=item_name, id=None, type=None)
```

Comment:
`advancement=item_name` sounds wrong. Shouldn't that be a boolean?

Reply:
it's what the other item_table assignment does, i wasn't about to start refactoring things

Reply:
It definitely does not? The other one says `HKItemData(advancement=item_name in logic_items or item_name in item_effects, id=i, type=item_type)`, which assigns `item_name in logic_items or item_name in item_effects` to `advancement`, so a boolean

Reply:
ah, read that wrong, pretty sure it works anyways because the HKItemData advancement is cast to a bool, but no reason to make it work harder than it already does